indexer/sql/src/entity/blocks.rs

```diff
@@ -52,6 +55,18 @@ impl Model {
         Ok(loader.load_one(self.clone()).await?.unwrap_or_default())
     }
 
+    async fn crons_outcomes(&self, ctx: &Context<'_>) -> Result<Vec<String>> {
+        Ok(BlockToIndex::load_from_disk(
```

Comment:
I'm not using it consistently but be aware since `load_from_disk` is blocking IO, it's better to use this in conjunction with `tokio::task::spawn_blocking(move || {`.

I should definitely change all the code whenever this is used.